Apps/W1/Shopify/test/Customer/ShpfyCreateCustomerTest.Codeunit.al

```diff
@@ -0,0 +1,84 @@
+/// <summary>
+/// Codeunit Shpfy Create Customer Test (ID 135604).
+/// </summary>
+codeunit 135604 "Shpfy Create Customer Test"
+{
+    Subtype = Test;
+    EventSubscriberInstance = Manual;
+
+    var
+        LibraryAssert: Codeunit "Library Assert";
+        ShpfyCommunicationMgt: Codeunit "Shpfy Communication Mgt.";
+#pragma warning disable AA0072
+        This: Codeunit "Shpfy Create Customer Test";
```

Comment:
This name doesn't look to be worth overriding the CodeCop warning. Languages that support 'this' or 'self' reference mean the current instance of an object, while this declaration creates a new instance. Why not just name it 'ShpfyCreateCustomerTest'?